client/src/components/annotators/README.md

```diff
@@ -0,0 +1,42 @@
+# annotator
+
+An annotator maintains an instance of GeoJS.
+
+## state
+
+The state available on the injected Vue annotator instance.
+
+```js
+viewer // geojs viewer
+playing // boolean
+frame // number
+maxFrame // number
```

Comment:
viewer - is there a better name we should be calling this object?  imageViewer, frameViewer, imageRenderer, frameRenderer or something like that.  Looking at the code my mind immediately goes 'there's a Viewer.vue, must have some relationship to that' and that turns out to be wrong.  Could be better to rename the Viewer.vue to something more fitting.

Reply:
It's an instance of `geo.map()` so something like `geoViewer` or `geoMap` I think would be appropriate.